wxmedit.txt

```diff
@@ -0,0 +1,59 @@
+[Section]
+Name = wxMEdit
+Category = 7
+URLDownload = http://downloads.sourceforge.net/project/wxmedit/3.2/wxMEdit-3.2-win32-bin-ros.zip
+SizeBytes = 16495754
+Description = A Cross-platform Text/Hex Editor
+Icon = wxmedit.ico
+License = GPLv3
+SHA1 = 316550af63f6f2eed217c602818ba89092b378a8
+URLSite = https://wxmedit.github.io/
+Version = 3.2
+Screenshot1 = https://github.com/reactos/rapps-db/assets/3896345/a7cb984d-1dcc-4db1-8e8d-2e390599bfe1
```

Comment:
I'd suggest to put it into category 6 like [notepad++](https://github.com/reactos/rapps-db/blob/master/npp.txt), also rearrange lines a bit

```suggestion
Name = wxMEdit
Version = 3.2
Description = A Cross-platform Text/Hex Editor
License = GPLv3
Category = 6
URLSite = https://wxmedit.github.io/
URLDownload = http://downloads.sourceforge.net/project/wxmedit/3.2/wxMEdit-3.2-win32-bin-ros.zip
SizeBytes = 16495754
SHA1 = 316550af63f6f2eed217c602818ba89092b378a8
Icon = wxmedit.ico
Screenshot1 = https://github.com/reactos/rapps-db/assets/3896345/a7cb984d-1dcc-4db1-8e8d-2e390599bfe1
```

Reply:
I don't care much about category 7 == development vs category 6 == office.
But thumbsUp for the reordering.

Reply:
Reordered.

Both the text editing and hexadecimal editing of wxMEdit are its main functions.
So I refer to Notepad2e, HxD & HexEdit, they are all with "Category = 7" (Development).
I think Category = 7 is more appropriate.